internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -0,0 +1,56 @@
+// Package codepipeline provides a client to make API requests to Amazon Elastic Container Service.
+package codepipeline
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	cp "github.com/aws/aws-sdk-go/service/codepipeline"
```

Comment:
Do we need to create this package alias?

Reply:
no but cp is easier to type =P